docusaurus/docs/getting-started.md

```diff
@@ -66,6 +66,44 @@ _`yarn create` is available in Yarn 0.25+_
 
 Follow our [Adding TypeScript](adding-typescript.md) documentation to create a TypeScript app.
 
+## Selecting a package manager
+
+When you run
+
+```sh
+npx create-react-app my-app
+```
+
+It by defaults uses `yarn` to install dependencies, but if you want to opt-in for
+`npm` then just append `--use-npm` to the script above like this;
```

Comment:
I'd suggest that we just simplify the above to skip both "When you run" and the example, and just say:
> When you create a new app, the CLI will use [Yarn](https://yarnpkg.com/) to install dependencies (when available). If you have Yarn installed, but would prefer to use npm, you can append `--use-npm` to the creation command. For example:

Reply:
I have made the suggested changes, can you kindly have a look at it again @mrmckeb